integrated-env

```diff
@@ -35,6 +36,8 @@ export CONTROL_REPO_HOST_PATH= ; echo "Please set CONTROL_REPO_HOST_PATH in ${BA
 # Set these to a git URL and branch to load the control repository from GitHub instead.
 # export CONTROL_REPO_URL=https://github.com/rackerlabs/nexus-control.git
 # export CONTROL_REPO_BRANCH=master
+export CONTROL_REPO_URL=
+export CONTROL_REPO_BRANCH=
 
```

Comment:
Maybe it's a mac thing but if these are not defined, when I attempt to execute `script/add-assets /Users/roge5001/Projects/RSDocs/nexus-control` I get the following error: `~integrated/env: line 81: CONTROL_REPO_URL: unbound variable`. These default values resolve that.

Reply:
I'd prefer to notify the user to uncomment and unset the variable rather than duplicating them. Also, you don't need to provide a blank `CONTROL_REPO_BRANCH` (note that I'm on a Mac); the following example is what's in my env with an updated comment:
```bash
# Set these to a git URL and branch to load the control repository from GitHub instead. If you are not loading from a remote repository, leave this alone.
export CONTROL_REPO_URL=
# export CONTROL_REPO_BRANCH=master
```